content/homework.rst

```diff
@@ -135,7 +167,7 @@ not received.
 Always acknowledge that you have received and understood a text message.
 Be as clear as possible and avoid subtle irony or local culture jokes.
 
-Since most of the communication is done using text, get proficient at typing.
+Since most of the communication is done using text, get proficient at typing and get good tools to help with typing (ie a good keyboard).
```

Comment:
+1 for this

Unfortunately this is the thing I hate on all laptops... the keyboards. Having to carry around a "real" keyboard makes it a hassle.